docs/data/data-grid/ai-assistant/AssistantWithDataSampling.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  unstable_gridDefaultPromptResolver as promptResolver,
+} from '@mui/x-data-grid-premium';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const VISIBLE_FIELDS = [
+  'id',
+  'avatar',
+  'name',
+  'website',
+  'rating',
+  'email',
+  'phone',
+  'username',
+  'position',
+  'company',
+  'salary',
+  'country',
+  'city',
+  'lastUpdated',
+  'dateCreated',
+  'isAdmin',
+];
+
+function processPrompt(prompt: string, context: string, conversationId?: string) {
+  return promptResolver(
+    'https://backend.mui.com/api/datagrid/prompt',
+    prompt,
+    context,
+    conversationId,
+  );
+}
+
+export default function AssistantWithDataSampling() {
+  const { data } = useDemoData({
+    dataSet: 'Employee',
```

Comment:
There's something wrong with row grouping by country in this dataset.
It's not caused by this PR though, I'll look into it

Reply:
Fixed by https://github.com/mui/mui-x/pull/17315